developer-docs-site/docs/nodes/ait/ait3-leaderboard-metrics.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "AIT-3 Leaderboard Metrics"
+slug: "ait-3-leaderboard-metrics"
+---
+
+# AIT-3 Leaderboard Metrics
+
+This document explains how the rewards for AIT-3 validator are evaluated and displayed on the [AIT-3 Validator Status](https://community.staging.gcp.aptosdev.com/leaderboard/it3?sort=-rewards_growth) page. 
+
+## How are rewards calculated
+
+:::tip Staking documentation
+For a backgrounder on staking with explanations of epoch, rewards and how to join and leave validator set, see the [Staking](/concepts/staking.md). 
+:::
+
+- For the AIT-3 the epoch duration is 7200 seconds, i.e., two hours.
```

Comment:
@movekevin , @sherry-x  - do we plan to have a governance proposal changing the epoch duration?